src/features/ajax/aggregate/index.js

```diff
@@ -119,6 +120,9 @@ export class Aggregate extends AggregateBase {
         event.spanTimestamp = xhrContext.dt.timestamp
       }
 
+      // parsed from the AJAX body, looking for operationName param & parsing query for operationType
+      event.gql = params.gql = parseBatchGQL(bodyString)
```

Comment:
```suggestion
      event.gql = parseBatchGQL(bodyString)
```

Reply:
params.gql can be id'd in the SPA feature, since it references the object context